README.md

```diff
@@ -2,77 +2,71 @@
 
 nesicide
 ========
+nesicide is an Integrated Development Environment (IDE) for the 8-bit Nintendo Entertainment System (NES).
 
-Integrated Development Environment for the 8-bit Nintendo Entertainment System
+This project contains:
+* NESICIDE (the IDE itself)
+* A NES emulator (standalone package of the emulator used in the IDE)
+* FamiTrackerQT (a fork of jsr's FamiTracker)
+* FamiPlayer (a music player for .ftm files)
 
 Prerequisites
 =============
-0. Install Qt open source from:
+To be able to build NESICIDE you need to do the following:
 
-   https://info.qt.io/download-qt-for-application-development
-   
-   NOTE: The latest kit that was verified to build NESICIDE properly with no errors is Qt 5.6.2. Please do not 
-         use a kit later than this, or if you do, please create a pull request with necessary changes. Known issues:
-         
-         . C++11 support required
+0. Install `git`
 
-1. Install git
+1. Install `Qt 5.6.2` [https://download.qt.io/official_releases/qt/5.6/5.6.2/](https://download.qt.io/official_releases/qt/5.6/5.6.2/) as it was the latest kit to be verified to build NESICIDE properly with not errors. Please do not use a kit later than this, or if you do, please create a pull request with nexessary changes. 
```

Comment:
nexessary?

Reply:
You still of course need to install the right QT version but the notice about the release might not be needed.

Reply:
Actually, should this be excluded altogether and replaced with an appropriate package manager command for the latest version of QT?

Reply:
Oh, I see. Haha. Spelling errors are hard to spot at times. :D